docker-compose.hobby.yml

```diff
@@ -307,6 +307,42 @@ services:
             - db
             - redis
 
+    cymbal:
+        image: ghcr.io/posthog/posthog/cymbal:master
+        build:
+            context: ./posthog/rust
+            args:
+                BIN: cymbal
+        restart: on-failure
+        volumes:
+            - ./share:/share
+        environment:
+            KAFKA_HOSTS: 'kafka:9092'
+            KAFKA_CONSUMER_GROUP: cymbal
+            KAFKA_CONSUMER_TOPIC: exceptions_ingestion
+            OBJECT_STORAGE_BUCKET: posthog
+            OBJECT_STORAGE_ACCESS_KEY_ID: 'object_storage_root_user'
+            OBJECT_STORAGE_SECRET_ACCESS_KEY: 'object_storage_root_password'
+            OBJECT_STORAGE_ENDPOINT: 'http://objectstorage:19000'
+            OBJECT_STORAGE_FORCE_PATH_STYLE: 'true'
```

Comment:
i think that's minio
might be interesting to switch to seaweedfs now
so that we don't need to migrate self-hosted in future

(it should _just_ work)

(cc @veryayskiy )

Reply:
Switched to seaweed and tested. All is working as expected. Cymbal correctly connects to seaweed now

Reply:
\o/